Makefile

```diff
@@ -242,6 +242,12 @@ bundle: manifests operator-sdk kustomize ## Generate bundle manifests and metada
 	$(KUSTOMIZE) build config/manifests | envsubst | $(OPERATOR_SDK) generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	$(OPERATOR_SDK) bundle validate ./bundle
 
+.PHONY: bundle-k8s
+bundle-k8s: bundle ## Generate bundle manifests and metadata customized to k8s community release, then validate generated files.
+	# Note that k8s 1.25+ needs PSA label
+	sed -r -i "s|by default\.|by default.\n    Note prior to installing SNR on a Kubernetes 1.25+ cluster, a user must manually set a privileged PSA label on SNR's namespace. It gives SNR's agents permissions to reboot the node (in case it needs to be remediated).|;" ./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml
```

Comment:
Maybe add a lnk to https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/ ?

Reply:
Sure.
I had it in the Original PR to our upstream docs (which I've copy pasted from) but it was dropped in the review process.